### ISSUE TITLE
Toe flexion creases hypoplastic or absent

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10427]
**Preferred term label:**
Toe flexion creases hypoplastic or absent

**Synonyms**

**Definition (free text, please give PubMed ID)**
UMLS has one term that is opposite:
[Deep skin creases in toes (C4014637)](https://uts.nlm.nih.gov/uts/umls/concept/C4014637)

UMLS also has some terms that are similar but less specific and highly overlapping:
[Absent flexion creases (C5775568)](https://uts.nlm.nih.gov/uts/umls/concept/C5775568)
[Hypoplastic flexion creases (C2677766)](https://uts.nlm.nih.gov/uts/umls/concept/C2677766)
[Decreased flexion creases (C4013275)](https://uts.nlm.nih.gov/uts/umls/concept/C4013275)
[Absence of flexion creases (C5543135)](https://uts.nlm.nih.gov/uts/umls/concept/C5543135)

**Parent term (use hpo.jax.org/app)**
HPO has some similar terms for fingers: https://hpo.jax.org/app/browse/term/HP:0006143

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/620662 "short halluces with small or absent toenails and absence of flexion creases"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000891
- label: Absent toe phalanx flexion crease
- parents: [Abnormal toe morphology (HP:0001780)](https://hpo.jax.org/app/browse/term/HP:0001780)
- pmid: 
- def: Absence of one or more interphalangeal creases of a toe (i.e., of the transverse lines in the skin between the phalanges of the toes).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10427
- synonyms:
